docs/paying-for-lit/overview.md

```diff
@@ -0,0 +1,129 @@
+# Paying for Usage of Lit
+
+The Lit networks, like other decentralized networks, manage their computational resources to ensure optimal performance and synchronization across their nodes. To maintain a responsive and balanced network, Lit implements a metering system for resource utilization.
```

Comment:
> The Lit networks, like other decentralized networks, manage their computational resources to ensure optimal performance and synchronization across their nodes. To maintain a responsive and balanced network, Lit implements a metering system for resource utilization.

IMO this wording is a little confusing. Maybe we can just say that the Lit network meters usage to ensure that optimal performance is maintained for all users, which is accomplished by allowing all users to reserve capacity on the network based on how much usage they require

Reply:
I reverted it back to:

Like other decentralized networks, Lit has a certain amount of computation available for users that's metered to allow for a responsive network with nodes that are able to stay in-sync with one another.